multi-arch/samples.md

```diff
@@ -68,11 +67,13 @@ Multi-arch has been declared a key initiative for v4.x.
 
 ### Goals
 
-x86 samples are not installed in a cluster with no x86 nodes.  Nor are any future non-x86 samples installed in a cluster with only x86 nodes.
+Install available samples for the current architecture. Do not install samples for other architectures while we 
+only support homogenous clusters, to avoid currently confusing error messages when an image for one architecture 
```

Comment:
homogeneous (you have this typo in a bunch of places i think...i know i make it a lot.  homogeneous also, where you have that)